eternalcore/src/main/java/com/eternalcode/core/command/argument/UserArgument.java

```diff
@@ -0,0 +1,54 @@
+package com.eternalcode.core.command.argument;
+
+import com.eternalcode.core.language.LanguageManager;
+import com.eternalcode.core.language.Messages;
+import com.eternalcode.core.user.User;
+import com.eternalcode.core.user.UserManager;
+import com.eternalcode.core.viewer.BukkitViewerProvider;
+import com.eternalcode.core.viewer.Viewer;
+import dev.rollczi.litecommands.argument.ArgumentName;
+import dev.rollczi.litecommands.argument.simple.OneArgument;
+import dev.rollczi.litecommands.command.LiteInvocation;
+import dev.rollczi.litecommands.command.sugesstion.Suggestion;
+import org.bukkit.Server;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.entity.Player;
+import panda.std.Option;
+import panda.std.Result;
+
+import java.util.List;
+
+@ArgumentName("player")
```

Comment:
player -> user?

Reply:
ale to jest dla graczy a nie dla programistów